optuna/study.py

```diff
@@ -175,6 +175,25 @@ def __setstate__(self, state: Dict[Any, Any]) -> None:
     def user_attrs(self) -> Dict[str, Any]:
         """Return user attributes.
 
+        Example:
+
+            .. testcode::
+
+                import optuna
+
+                def objective(trial):
+                    x = trial.suggest_float("x", 0, 1)
+                    y = trial.suggest_float("y", 0, 1)
+                    return x ** 2 + y ** 2
+
+                study = optuna.create_study()
+
+                study.set_user_attr("objective function", "quadratic fuction")
+                study.set_user_attr("dimensions", 2)
+                study.set_user_attr("contributors", ["Akiba", "Sano"])
+
+                study.user_attrs
```

Comment:
How about checking the contents of the `study.user_attrs`?
```suggestion
                assert study.user_attrs == {
                    "objective function": "quadratic function",
                    "dimensions": 2,
                    "contributors": ["Akiba", "Sano"]
                }
```

Reply:
Nice idea!